app/controllers/bank_accounts_controller.rb

```diff
@@ -22,15 +28,27 @@ def create
     @bank_account = current_organization.bank_accounts.build(bank_account_params)
 
     if @bank_account.save
-      redirect_to bank_accounts_path, notice: 'Bank account was successfully created.'
+      respond_to do |format|
+        format.html do
+          redirect_to bank_accounts_path, notice: 'Bank account
```

Comment:
@dissident In future try to use resource style urls. So instead using named url helpers like `bank_accounts_path` use as you pass params to `url_for` method: `:bank_accounts`